app/javascript/dashboard/components/widgets/WootWriter/ReplyBottomPanel.vue

```diff
@@ -257,6 +257,9 @@ export default {
       return this.showAudioRecorder && this.isRecordingAudio;
     },
     allowedFileTypes() {
+      if (this.isATwilioWhatsappChannel) {
+        return ALLOWED_FILE_TYPES.replace('text/csv, text/plain, ', '');
```

Comment:
@iamsivin I believe it would be better to create a separate constant for Twilio with the supported items only.

Reply:
Not all video, audio or images are not supported.

Reply:
Okay, will update it.

Reply:
Done. Can you check?